DataEngineering/sc.py

```diff
@@ -0,0 +1,36 @@
+import json, requests
+import sqlite3
+
+
+conn = sqlite3.connect('Bike.db')
+c = conn.cursor()
+c.execute('CREATE TABLE if not exists realtimewithT (Id,value INTEGER, timestamp)')
+c.execute('CREATE TABLE if not exists realtime (id,location, lat FLOAT, lon FLOAT)')
+conn.commit()
+
+
+url = "https://api.tfl.gov.uk/bikepoint"
+response = requests.get(url)
+data = response.text
+parsed = json.loads(data)
+#print (parsed)
+
+rn= (len(parsed))
+for i in range(rn):

```

Comment:
Where is i being used? You are only storing data from the 0th element here - what about the rest of the bikestands?